lib/IDE/CodeCompletion.cpp

```diff
@@ -437,29 +445,32 @@ void CodeCompletionString::print(raw_ostream &OS) const {
       break;
     case ChunkKind::OptionalBegin:
     case ChunkKind::CallArgumentBegin:
-    case ChunkKind::CallArgumentTypeBegin:
```

Comment:
Why does this not get '{##}' anymore? Is it handled somewhere else?

Reply:
We don't have tests for this case.
`ChunkKind::CallArgumentTypeBegin` is only for "annotated" results. For`swift-ide-test -code-completion -code-completion-annotate-results`, we test `printCodeCompletionResultTypeNameAnnotated()`, but not `CodeCompletionString::print()`.

I moved this to "not enclosed" group because, in future, when we unify `CallArgumentTypeBegin` and `CallArgumentType` chunk kinds, since `CallArgumentType` is not enclosed by `{##}`, enclosing `CallArgumentTypeBegin` with it would cause massive test case changes.